docs/melange-book/index.md

```diff
@@ -0,0 +1,16 @@
+# Melange Book
+
+## Motivation
+
+This is a project-based, guided introduction to Melange and its ecosystem.
+Because Melange uses both OCaml and JavaScript ecosystems, there are a lot of
+tools and concepts to learn, which can be overwhelming for newcomers. Therefore
+we try to make each chapter small and digestible, not introducing too many
+things at once.
+
+## Audience
```

Comment:
Wonder if we should mention somewhere that because the audience target are React devs, the book will use Reason syntax all along?